msteams-platform/resources/schema/localization-schema.md

```diff
@@ -8,18 +8,18 @@ ms.date: 05/20/2019
 
 # Localize JSON schema reference
 
-The Microsoft Teams localization file describes language translations that are served based on the client language settings. Your file must conform to the schema hosted at [`https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.Localization.schema.json`](https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.Localization.schema.json).
+The Microsoft Teams localization file describes language translations that are served based on the client language settings. Your file must conform to the schema hosted at [`https://developer.microsoft.com/en-us/json-schemas/teams/v1.17/MicrosoftTeams.Localization.schema.json`](https://developer.microsoft.com/en-us/json-schemas/teams/v1.17/MicrosoftTeams.Localization.schema.json).
 
 > [!TIP]
-> Specify the schema at the beginning of your manifest to enable `IntelliSense` or similar support from your code editor: `"$schema": "<https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json>".
+> Specify the schema at the beginning of your manifest to enable `IntelliSense` or similar support from your code editor: `"$schema": "<https://developer.microsoft.com/json-schemas/teams/v1.17/MicrosoftTeams.schema.json>".
 
 ## Example
```

Comment:
```suggestion
## Localization schema v1.17
```

Could you please change this? Looking at "In this Article" section, developer might just navigate to 1.16 schema version. 
![image](https://github.com/user-attachments/assets/a607bea9-9d4d-4ee0-9991-cc6186074a43)